src/collective/collectionfilter/upgrades.py

```diff
@@ -98,3 +98,7 @@ def upgrade_to_plone6(context):
             del registry.records[key]
 
     reapply_profile(context)
+
+def upgrade_registry_13(context):
```

Comment:
If you only want to reload an existing `import_step` I'd not do this in python code but in `zcml` see my change suggestion below.